Source/WebCore/loader/FrameLoader.h

```diff
@@ -543,6 +547,7 @@ class FrameLoader final : public CanMakeWeakPtr<FrameLoader> {
 
     bool m_errorOccurredInLoading { false };
     bool m_doNotAbortNavigationAPI { false };
+    bool m_isHandledByAboutSchemeHandler { false };
```

Comment:
I doubt this is the right location to be storing this flag. Normally, flags that have to do with one particular navigation are stored on the DocumentLoader, not the FrameLoader. The reason is that the FrameLoader gets re-used for every navigation, while the DocumentLoader is per navigation.

Reply:
You may want to look at the `DocumentLoader::m_isRequestFromClientOrUserInput` flag for example, I think it can also come from the UIProcess like your new flag.